src/PowerShellEditorServices/Services/Symbols/SymbolDetails.cs

```diff
@@ -34,6 +34,7 @@ internal class SymbolDetails
 
         #region Constructors
 
+        // TODO: This should take a cancellation token!
```

Comment:
Whereas this one I think could actually be done and I meant to add a commit to do it.